draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -994,16 +1100,21 @@
 
-      <t>This behavior is analogous to the impacts observed with RSVP auto-bandwidth, which can introduce considerable instability within a network. Such route churn can propagate through the network, causing a cascade of updates and potential route flaps, thereby affecting overall network stability and performance.</t>
+      <t>This behavior is analogous to the impacts observed with RSVP auto-bandwidth, which can introduce considerable instability within a network. Such route churn can propagate through the network, causing a cascade of UPDATEs and potential route flaps, thereby affecting overall network stability and performance.</t>
 
-	  <t>To mitigate these effects, network operators should carefully manage the advertisement intervals of these dynamic metrics, ensuring they are set to avoid unnecessary churn. The default minimum interval for metrics change advertisement, set at 30 seconds, is designed to balance responsiveness with stability. However, in scenarios with higher sensitivity to route stability, operators may consider increasing this interval further to reduce the frequency of updates.</t>
+	  <t>To mitigate these effects, network operators should carefully manage the advertisement intervals of these dynamic metrics, ensuring they are set to avoid unnecessary churn. The default minimum interval for metrics change advertisement, set at 30 seconds, is designed to balance responsiveness with stability. However, in scenarios with higher sensitivity to route stability, operators may consider increasing this interval further to reduce the frequency of UPDATEs.</t>
 
 	  <t> Furthermore, operators should implement robust route damping and filtering policies to control the propagation of changes and minimize the impact on dependent routes. By acknowledging and planning for these broader impacts, the mechanism can be deployed more effectively, ensuring optimal performance without compromising network stability.</t>
+
+	  <!-- XXX JMH - You don't really mean "damping" here.  Route flap
+	  damping responds to high degrees of churn by suppressing the prefix
+	  (withdraw).  What's your intent here? -->
       
 	  <t>Significant load changes at EC data centers can be triggered by
       short-term gatherings of UEs, like conventions, lasting a few hours or
       days. Therefore, the load metrics change rate can be in the
       magnitude of hours or days.</t>
+      <!-- XXX JMH - Do you mean that the high change range can last for this period of time? -->
```

Comment:
yes. Meaning the change rate is in days (not in minutes)

Reply:
I've tweaked the text. See next push.